src/components/LandingPage/index.tsx

```diff
@@ -44,6 +45,10 @@ const LandingPage = ({ handleSelectDataset, megasets }: LandingPageProps) => (
                             ))}
                         </Row>
                     </div>
+
+                    <div className={styles.sectionContent}>
+                        <CsvUpload />
```

Comment:
I think this should be called CsvDrop or something. you're not uploading it to an external source